README.md

```diff
@@ -25,66 +25,157 @@ filter:
 
 show_empty: <show_empty>
-else: <else>
 unique: <unique>
 sort: <sort_method>
 ```
 
-| Option                 | Type                             | Description                                                                                                           | Default         |
-| ---------------------- | -------------------------------- | --------------------------------------------------------------------------------------------------------------------- | --------------- |
-| `card`                 | Dashboard card\*                 | The card to display. Specify this as you would specify any normal dashboard card, but ommit the `entities` parameter. | `entities`-card |
-| `entities`             | List of Entities\*\*             | Any entities added here will be added to the card before any filters are applied                                      |
-| `filter`               |
-| &nbsp;&nbsp;`template` | string                           | A jinja template evaluating to a list of entities to include                                                          |
-| &nbsp;&nbsp;`include`  | List of [Filters](#filters)      | A list of filters specifying which entities to add to the card                                                        |
-| &nbsp;&nbsp;`exclude`  | List of [Filters](#filters)      | A list of filters specifying which entities to remove from the card                                                   |
-| `show_empty`           | `true`/`false`                   | Whether to display the card if there are no entities                                                                  | `false`         |
-| `else`                 | Dashboard card\*                 | Card to display if the main card has no entities. Overrides `show_empty`                                              |
-| `sort`                 | [Sort config](#sorting-entities) | How to sort the entities of the card                                                                                  | `none`          |
-| `card_param`           | string                           | The parameter of the card to populate with entities                                                                   | `entities`      |
-
-\* [Dashboard card](https://www.home-assistant.io/dashboards/cards/) \
-\*\* [Entities](https://www.home-assistant.io/dashboards/entities/#options-for-entities)
+## Options
+
+- `card:` **Required.** The card to display. Specify this as you would specify any normal lovelace card, but ommit the `entities:` parameter.
+- `entities:` Any entities added here will be added to the card before any filters are applied.
+- `filter:`
+  - `template:` A jinja2 template evaluating to a list of entries to include
+  - `include:` A list of filters specifying which entities to add to the card
+  - `exclude:` A list of filters specifying which entities to remove from the card
+- `names:`
+  - `transforms:` A list of transforms to be applied to all entity names
```

Comment:
What if a card does not support `name`? This would not work at all? It should be stated both that this option passes `name` to the card like it does `entity`, so only works if card supports `name` overrides, but otherwise will do nothing and may cause issues if card does config validation and does not support name.

Also, suggest to use singular `name` and `transform` which matches using singular for `card`, `filter` etc.